server/routes/__tests__/comments.test.js

```diff
@@ -3,26 +3,44 @@ const server = require('../../server')
 const db = require('../../db')
 
 import { objComment } from '~/test/fake-data'
+const { checkJwt } = require('../../utils')
 
 jest.mock('../../db')
+jest.mock('../../utils')
+
+beforeAll(() => {
+  jest.spyOn(console, 'log')
+  console.log.mockImplementation(() => {})
+  checkJwt.mockImplementation((req, res, next) => {
+    next()
+  })
```

Comment:
this mockImplementation needs to add something to req.user so you can test that the auth0Id was passed down to `db.insertComment` correctly